yarn.lock

```diff
@@ -19916,6 +19914,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"type-fest@npm:^0.21.3":
```

Comment:
this isn't now shipped right? comes from some dev dep?

Reply:
Right. For instance `lerna-lite` ships same old `ansi-escapes` through `inquirer`. They cannot update `inquirer`, because newer version is ESM only.

Reply:
ESM only... gotta love it 😅